dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/trackedentity/DefaultTrackedEntityService.java

```diff
@@ -340,27 +340,16 @@ public TrackedEntities getTrackedEntities(TrackedEntityOperationParams operation
     return TrackedEntities.of(trackedEntities, pager);
   }
 
-  public List<Long> getTrackedEntityIds(
-      TrackedEntityQueryParams params,
-      boolean skipAccessValidation,
-      boolean skipSearchScopeValidation) {
+  public List<Long> getTrackedEntityIds(TrackedEntityQueryParams params) {
     if (!params.hasProgram()) {
       Collection<TrackedEntityAttribute> attributes =
           trackedEntityAttributeService.getTrackedEntityAttributesDisplayInListNoProgram();
       attributes.forEach(params::filterBy);
     }
 
     decideAccess(params);
-
-    // AccessValidation should be skipped only and only if it is internal
-    // service that runs the task (for example sync job)
```

Comment:
Why don't we need the `if`s anymore?

Reply:
This method was only called in one place and the checked flags were statically passed as false